### DISCUSSION TITLE
Is internet connectivity required, or is this a bug?

### DISCUSSION THREAD

**Discussion by icmccorm** [openai/whisper#1296] [Q&A] [answered]
I've been using Whisper for a few weeks now to conduct interview transcriptions offline, with the assumption from #1100 that raw audio is never uploaded to OpenAI. However, recently whisper crashed with the following error message while running offline:
```
/Users/.../.pyenv/versions/3.9.9/lib/python3.9/site-packages/whisper/transcribe.py:114: UserWarning: FP16 is not supported on CPU; using FP32 instead
  warnings.warn("FP16 is not supported on CPU; using FP32 instead")
Detecting language using up to the first 30 seconds. Use `--language` to specify the language
Traceback (most recent call last):
  File "/Users/.../.pyenv/versions/3.9.9/lib/python3.9/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/Users/.../.pyenv/versions/3.9.9/lib/python3.9/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/Users/.../.pyenv/versions/3.9.9/lib/python3.9/socket.py", line 954, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known
```
After I reconnected to the internet, subsequent attempts worked fine, both offline and online. Is this a bug? Or is a regular connection required to use Whisper?

**Comment by jongwook** [comment, marked as answer]
It might have been caused by tiktoken trying to download the GPT2 tokenizer, which wasn't necessary and fixed in #1163. I haven't yet made a release after that, but can you try with the latest commit and see if the issue continues?

**Reply by icmccorm** [reply]
I haven't been able to recreate the issue, but I'll update this thread if I encounter it again!
